.github/workflows/scripts/post_before_script.sh

```diff
@@ -2,13 +2,11 @@
 
 set -euv
 
-echo "machine pulp
-login admin
-password password
-" > ~/.netrc
+# Add the QE (public) key to the container env for `test_sign_locally_then_upload_signature`.
+# Please, remove once the verification was made independent of the system keyring.
 
-chmod og-rw ~/.netrc
+cmd_prefix bash -c "curl -L https://github.com/pulp/pulp-fixtures/raw/master/common/GPG-KEY-pulp-qe | gpg --import"
```

Comment:
Thanks. With the fixtures from pulpcore I don't think this is needed. I'm sure I'm not understand it. I expected all keyring config to not be global and to be provided from the pulpcore fixtures.

Reply:
Basically the signature upload verification code is bugged right now, it doesn't work if a keyring isn't specified and the key is not in the global (default) keyring. This is why the test passed on my machine, but not on the CI. @mdellweg is creating a prototype replacement to fix this.